lib/suspenders/app_builder.rb

```diff
@@ -163,8 +163,8 @@ def replace_gemfile
       template 'Gemfile.erb', 'Gemfile'
     end
 
-    def set_ruby_to_version_being_used
-      create_file '.ruby-version', "#{Suspenders::RUBY_VERSION}\n"
+    def set_ruby_to_latest_version
+      create_file '.ruby-version', "#{Suspenders::LATEST_RUBY_VERSION}\n"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.